src/java/org/apache/cassandra/db/commitlog/CommitLog.java

```diff
@@ -129,6 +129,10 @@ private static CommitLog construct()
 
         // register metrics
         metrics.attach(executor, segmentManager);
+
+	if (configuration.isDirectIOEnabled()) {
```

Comment:
I think we may do not need this logging ~~~ or just  logger.info("Use direcrtor io for commit log " +  configuration.isDirectIOEnabled())